src/main/java/net/minecraftforge/fml/relauncher/CoreModManager.java

```diff
@@ -216,6 +218,16 @@ public static void handleLaunch(File mcDir, LaunchClassLoader classLoader, FMLTw
         {
             FMLLog.log.debug("Enabling runtime deobfuscation");
         }
+        else
+        {
+            if (System.getProperty("log4j.configurationFile") == null)
```

Comment:
Still needed here, but in my original PR I added this additionally in `ServerLaunchWrapper.java`. That way it is set early enough so we don't need to reconfigure the logging.

See: https://github.com/MinecraftForge/MinecraftForge/pull/3965/files#diff-d4776de6664b055501dd17ee4290e477R41

Reply:
Good catch, I thought I had included that too.

Reply:
The stuff used in the server config has detection of the terminals's ability to print colored text, I expect that Eclipse and Netbeans will not see a difference. That said, I tested several terminals but I haven't tested IDEs other than IntelliJ.